### PR TITLE
tag users in the question_list

### PR DESCRIPTION
**Pull request by stelar7**
Added since !ask and !ask_for stores different data in the list..

Optionally should !ask be changed to tag? as usernames might be different from the displayname..

### CODE REVIEW

**Comment by Querijn** [comment]
Why not? If we cannot provide an answer, and we're going to ask Riot for it, they would probably like to be notified of the answer once they get it..

**Comment by stelar7** [comment]
We should have !ask and !ask_for have the same interaction with the other commands imo. and as its now, one of them tags, the other doesnt.

**Comment by Querijn** [comment]
I'm kind of confused where this PR is at right now, but basically, I need the following behaviour:
1. *Don't* mention users in !question_list, so that if it is used outside of a moderator channel, all the users don't get a notification
2. !ask and !ask_for I don't really care about, as long as they're consistent (both mention or neither mention)
3. The listing of users will be tagged during office hours.

**Comment by Querijn** [comment]
At the least, this one needs to be fixed: 
afbeelding

**Comment by stelar7** [comment]
As it is now, both !ask and !ask_for should add the mention to the datastore, if you dont want !question_list to tag users, we need to fetch users->username in that command.

File: src/OfficeHours.ts [hunk 1]
```diff
@@ -61,15 +61,15 @@ export default class OfficeHours {
 
     public onAsk(message: Discord.Message, isAdmin: boolean, command: string, args: string[]) {
         const question = args.join(" ");
-        this.storeQuestion(question, message, message.author.id, message.author.toString());
+        this.storeQuestion(question, message, message.author.toString(), message.author.username);
     }
 
     public onAskFor(message: Discord.Message, isAdmin: boolean, command: string, args: string[]) {
         const asked = message.mentions.members.first();
         if (!asked) return;
```
Comment (Querijn) [inline src/OfficeHours.ts]: This makes sense in this boolean-esque situation, but is weird below (asked.id and asked.toString()). Can you change it to `asker`?

**Pull request merged**